api/database/migrations/2021_08_31_130347_update_pools_table_id_type.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdatePoolsTableIdType extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement('ALTER TABLE "pools" ALTER COLUMN id TYPE text');
```

Comment:
Is there a reason that the Database query builder was used here instead of the [Schema facade to update the column type](https://laravel.com/docs/8.x/migrations#updating-column-attributes)?

Reply:
No, just the first thing that came up mind. If we want to be strict on the style of syntax we use, happy to swap it over. Just wasn't aware know we had any standard on that

Reply:
Yeah, for consistency we should use the schema facade if there's a method for this.

Reply:
Okay, will do.